### ISSUE TITLE
Firebase Deploy Fails

### ISSUE THREAD

**Issue by ackushiw** [firebase/firebase-tools#202]
I just updated to version 3.0.6
I've set the project up with only rules and hosting.

using  `firebase deploy` returns the following error:

```
[debug] TypeError: Cannot read property 'triggers' of undefined
    at module.exports (/usr/local/lib/node_modules/firebase-tools/lib/deploy/functions/release.js:155:40)
    at _chain (/usr/local/lib/node_modules/firebase-tools/lib/deploy/index.js:21:45)
    at /usr/local/lib/node_modules/firebase-tools/lib/deploy/index.js:77:12
    at lib$rsvp$$internal$$tryCatch (/usr/local/lib/node_modules/firebase-tools/node_modules/rsvp/dist/rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (/usr/local/lib/node_modules/firebase-tools/node_modules/rsvp/dist/rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (/usr/local/lib/node_modules/firebase-tools/node_modules/rsvp/dist/rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (/usr/local/lib/node_modules/firebase-tools/node_modules/rsvp/dist/rsvp.js:1198:9)
    at nextTickCallbackWith0Args (node.js:456:9)
    at process._tickCallback (node.js:385:13)
[error] 
[error] Error: An unexpected error has occurred.
```

using `firebase deploy --only hosting` works fine.

**Comment by mbleigh** [comment]
This should be fixed in the just-released `3.0.7` -- can you give it a try?

    npm install -g firebase-tools

**Comment by ackushiw** [comment]
Works! Thanks!
